Xamarin.Forms.Platform.iOS/Renderers/EditorRenderer.cs

```diff
@@ -181,5 +184,10 @@ bool ShouldChangeText(UITextView textView, NSRange range, string text)
 			var newLength = textView.Text.Length + text.Length - range.Length;
 			return newLength <= Element.MaxLength;
 		}
+
+		void UpdateIsReadOnly()
+		{
+			Control.UserInteractionEnabled = !Element.IsReadOnly;
```

Comment:
Just an FYI that `UpdateEditable` is also changing `UserInteractionEnabled`. If I set `IsEnabled` to false, then the control becomes read-only even though `IsReadOnly` is false.

Reply:
Also, please look at other renderers to see if they are doing similar work. We should make sure `IsEnabled` and `IsReadOnly` play nice together.

Reply:
Nice point, thanks! :) Will fix that

Reply:
"If I set IsEnabled to false, then the control becomes read-only even though IsReadOnly is false.".

Yes, the main task of this Enhancement is to make control ReadOnly without changing the look of it, so if you set IsEnabled to false it will be "read-only" and also it will have a look like it is disabled. But if you are using only IsReadOnly = true, it will become ReadOnly and the look will not change. 

So, maybe the best approach is to set IsReadOnly to true if IsEnabled == false ? :)

Reply:
Change UpdateIsReadOnly() to UpdateUserInteraction()

In UpdateEditable, change the assignment of UserInteractionEnabled to point at this function instead.

Then in this function, do
Control.UserInteractionEnabled = !Element.IsReadonly && Element.IsEnabled

I think that should work, but you get the gist. The change need to account for both properties now.

Reply:
Thanks!

Reply:
Yap @adamped suggestion is the correct one, we should have 1 method to handle this, the others call this.

Reply:
Ok :)